doc/development/api/virtual_schema_api.md

```diff
@@ -535,9 +547,6 @@ This document is usually embedded in responses from the Adapter and informs the
 }}
 ```
 
-Notes
-* `adapterNotes` is an optional field which can be attached to the schema, a table or a column. It can be an arbitrarily nested Json document.
```

Comment:
I would keep the word "optional" somewhere.

Reply:
Added above.